README.md

```diff
@@ -8,22 +8,24 @@ If you have any issues using the API or would like some help, please open [an is
 
 ## API Overview
 
-The API tends to be RESTful and returns results in JSON by default.
+The API is generally RESTful and returns results in JSON by default.
```

Comment:
@tsgit commented:
>what does "tends to be" mean ?

I mean it strives to respect RESTful guidelines. But I'm not sure it's completely well-defined. I've used a different weasel word here.